packages/core/src/payment/strategies/braintree/braintree-payment-options.ts

```diff
@@ -173,6 +173,7 @@ export type BraintreeFormFieldStyles = Partial<
 
 export interface BraintreeFormFieldKeyboardEventData {
     fieldType: string;
+    errors?: any;
```

Comment:
Is it possible to update the `errors` interface?
As far as I see, `errors` should be an object with optional keys and values inside. We know the basic shape of this param and can create another interface for that.

Base on your comment we can create something like this:
```
export interface BraintreeFormError {
    isEmpty: boolean;
    isPotentialyValid: boolean;
    isValid: boolean;
}

export interface BraintreeFormErrorsData {
    number?: BraintreeFormError;
    expirationDate?: BraintreeFormError;
    expirationMonth?: BraintreeFormError;
    expirationYear?: BraintreeFormError;
    cvv?: BraintreeFormError;
    postalCode?: BraintreeFormError;
}
```

Reply:
Thanks for the advice!